### DISCUSSION TITLE
Y 축 AXIS 관련

### DISCUSSION THREAD

**Discussion by sayKorea** [nhn/tui.chart#743] [Q&A]
안녕하세요.
X축 데이터가 많은 상태( 3년치)에서 스크롤을 생성해서 옆으로 보여 주고 싶습니다.
Y Axis Area을 고정 할 수 있는 방법이 있을까요?

**Comment by jwlee1108** [comment]
@sayKorea 

> You need to leave a question in English for others. it is a kind of contribution to this repository.

## Translated
I want to create a scroll in a lot of X -axis data (3 years) and show it sideways. Is there a way to fix Y AXIS Area?

---

Can you show a more detailed example? Doesn't the problem solve by setting height and width?
